packages/app/src/app/pages/common/Navigation/index.js

```diff
@@ -94,10 +94,12 @@ function Navigation({ signals, store, title, searchNoInput }) {
                   style={{ placement: 'relative', fontSize: '1.25rem' }}
                   onClick={open}
                 >
-                  <BellIcon height={35} />
-                  {store.userNotifications.unreadCount > 0 && (
-                    <UnreadIcon count={store.userNotifications.unreadCount} />
-                  )}
+                  <Tooltip placement="bottom" content="Show Notifications">
```

Comment:
How about the content is based on the `unreadCount` that is if `unreadCount` is 0 it tells `No Notifications`. Thoughts? @Jensderond

Reply:
Something like this?

```suggestion
                  <Tooltip placement="bottom" content={store.userNotifications.unreadCount > 0 ? 'Show Notifications' : 'No Notifications'}>
```

Reply:
What do you think @SaraVieira?

Reply:
Looks good! Can you please commit your own suggestion? I don't have permission :(